pkg/sql/session_migration_test.go

```diff
@@ -209,6 +210,30 @@ WHERE dump.variable IS NULL OR dump2.variable IS NULL OR dump.variable != dump2.
 					return err.Error()
 				}
 				return ret
+
+			case "error":
+				var errorRE string
+				for _, arg := range d.CmdArgs {
+					if arg.Key == "regexp" {
+						if len(arg.Vals) != 1 {
+							t.Fatalf("regexp arg expects one value")
+						}
+						errorRE = arg.Vals[0]
+					}
+				}
+				if errorRE == "" {
+					t.Fatalf("error requires regexp arg")
+				}
+				_, err := dbConn.Exec(ctx, getQuery())
+				if err != nil {
+					if ok, err := regexp.MatchString(errorRE, err.Error()); ok {
+						return "error matched successfully"
```

Comment:
nit: i think it'd be simpler to return an empty string here, but not a blocking comment.

Reply:
:+1: